source/adminguide/storage.rst

```diff
@@ -661,6 +661,37 @@ The following table shows possible combinations of Service offering supported re
    Shrinking the Root disk is not supported via the service offering resizing workflow. All the combinations above assume a transition to Root disks with size equals or bigger than the original.
    Service Offerings with Root size of 0GB do not change the disk size to Zero and indicates that the offering do not enforces a Root disk size.
 
+Change disk offering for volume
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are volume operations like migrate volume and resize volume and both accepts new disk offering to replace the existing disk offering of volume.
+Instead of those APIs, Disk offering of a volume can be changed directly using changeOfferingForVolume API, also supported from UI.
```

Comment:
```suggestion
Instead of those APIs, it can be performed directly using change offering for volume operation.
```
I think, no need to explicitly mention supported from UI (as the doc is primarily with respect to UI)

Reply:
```suggestion
Instead of using these APIs directly, the operation can be performed in the UI using change offering in the details view for the volume.
```

Reply:
I've updated with Daan's version.